src/utils/iqeEnablement.test.ts

```
@@ -0,0 +1,196 @@
+import { describe, test, expect, jest, beforeEach, afterEach } from '@jest/globals';
```

ESLint

[error] Member 'expect' of the import declaration should be sorted alphabetically.

(sort-imports)